python/tester/upload.py

```diff
@@ -94,12 +95,12 @@ def send_next_visit(publisher, group, visit_infos):
         represent multiple snaps.
     """
     _log.info(f"Sending next_visit for group: {group}")
-    topic_path = publisher.topic_path(PROJECT_ID, "nextVisit")
+    topic = "nextVisit"
```

Comment:
Is this still the correct topic name? The [`overlays/dev`](https://github.com/slaclab/rubin-usdf-prompt-processing/blob/main/kubernetes/overlays/dev/kafka/next-visit-topic.yaml) uses `"next-visit-topic"`, and we may want separate topics for the real-world and development environments.

Reply:
Or should it be `pp-bucket-notify-topic` from the [activator|https://github.com/lsst-dm/prompt_prototype/blob/main/python/activator/activator.py#L60] if I read that correctly? 
Yeah we probably want to decide what topics to use for real-world and dev for all components?

Reply:
I agree that the same question applies to `pp-bucket-notify-topic`, but we probably shouldn't use it for `next_visit` in addition to image arrival notifications.

Reply:
I'm changing this to `next-visit-topic` for now. When we find out what name Summit really uses it can be changed easily.